js/index.js

```diff
@@ -1,6 +1,39 @@
 // Iteration 1: Names and Input
-//
+var hacker1 = "Duarte";
+console.log(`The driver name is ${hacker1}` );
+var hacker2 = "Alexandre";
+console.log(`The navigator name is ${hacker2}`);
```

Comment:
Try and use let/const instead of var. It still works but nowadays as a rule of thumb we try to use let and const